src/pages/Thanks/ThanksPage.vue

```diff
@@ -473,6 +493,31 @@ export default {
 			fragment: experimentVersionFragment,
 		}) || {};
 		this.enableMayChallengeHeader = shareChallengeExpData?.version === 'b';
+
+		// Only show the old page if the first page is a borrower page and feature flag is enabled
+		const newThanksPageEnabled = data?.general?.newThanksPageEnabled?.value === 'true';
+
+		if (newThanksPageEnabled && this.$appConfig?.firstPage?.indexOf('/lend/') !== -1) {
```

Comment:
The comment mentions only showing the new thanks page if the first page of the session is the BP, but this would also target the `/lend/filter` page, which I also think is a common landing page for users. You could do a regex match here and make sure the match is /lend/number.